ibm/service/appconfiguration/resource_ibm_app_config_feature.go

```diff
@@ -161,7 +171,7 @@ func resourceIbmIbmAppConfigFeatureCreate(d *schema.ResourceData, meta interface
 	options.SetEnabledValue(d.Get("enabled_value").(string))
 	options.SetEnvironmentID(d.Get("environment_id").(string))
 	options.SetDisabledValue(d.Get("disabled_value").(string))
-
+	options.SetRolloutPercentage(int64(d.Get("rollout_percentage").(int)))
```

Comment:
rollout_percentage is optional so you need to call d.GetOk(...) and then set the value if its defined

Reply:
```
if _, ok := d.GetOk("rollout_percentage"); ok {
			options.SetRolloutPercentage(int64(d.Get("rollout_percentage").(int)))
		}
		```

Reply:
Addressed the above comments